tests/mix/testconn.nim

```diff
@@ -174,3 +175,82 @@ suite "Mix Protocol":
     await conn.close()
 
     check response != 0.seconds
+
+  asyncTest "length-prefixed protocol - verify readLp fix":
+    ## This test verifies the fix for the length prefix bug where responses
+    ## from protocols using readLp() were losing their length prefix when
+    ## flowing back through the mix network.
+    const TestCodec = "/lengthprefix/test/1.0.0"
+
+    # Test message that will be sent and received
+    let testMessage = "Hello from mix protocol with length prefix!"
```

Comment:
testMessage - would be fancy to use some message that inspires Logos movement, network states, cypherpunk's manifesto or anything that comes to your mind in similar context... this can be your personal message or quote from the book (eg. farewell to westphalia) that will be be your _message in the code_.

Reply:
https://github.com/vacp2p/nim-libp2p/blob/cf0d80947561a256b30068eb95bbd6e4c7012f9c/tests/transports/connection_tests.nim#L17

we already started this practice here